compair/static/modules/assignment/assignment-module.js

```diff
@@ -1270,7 +1376,9 @@ module.controller("AssignmentWriteController",
             'astart': {'date': new Date(), 'time': new Date().setHours(0, 0, 0, 0)},
             'aend': {'date': new Date(), 'time': new Date().setHours(23, 59, 0, 0)},
             'cstart': {'date': new Date(), 'time': new Date().setHours(23, 59, 0, 0)},
-            'cend': {'date': new Date(), 'time': new Date().setHours(23, 59, 0, 0)}
+            'cend': {'date': new Date(), 'time': new Date().setHours(23, 59, 0, 0)},
+            'sestart': {'date': new Date(), 'time': new Date().setHours(23, 59, 0, 0)},
```

Comment:
Can you change the time to `new Date().setHours(0, 0, 0, 0)` for `sestart` and `cstart` (this can close #793)